react/features/settings/functions.js

```diff
@@ -83,14 +86,12 @@ export function getMoreTabProps(stateful: Object | Function) {
     } = state['features/base/conference'];
     const followMeActive = Boolean(state['features/follow-me'].moderator);
     const configuredTabs = interfaceConfig.SETTINGS_SECTIONS || [];
-    const localParticipant = getLocalParticipant(state);
-
 
     // The settings sections to display.
-    const showModeratorSettings = Boolean(
-        conference
+    const showModeratorSettings
```

Comment:
Isn't it good to keep the `Boolean(...)`?

Reply:
Yes you're right.